lib/datadog/tracing/distributed/trace_context.rb

```diff
@@ -63,6 +63,7 @@ def extract(data)
             trace_flags: trace_flags,
             trace_state: tracestate,
             trace_state_unknown_fields: unknown_fields,
+            trace_remote: true,
```

Comment:
Should we name this field span_remote?

Reply:
`span_id` should have been `parent_id`...

Let's change it to `span_remote` then.